model/Security/Properties/decisionType.md

```diff
@@ -4,11 +4,13 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Provide the enumeration of possible decisions in the Stakeholder-Specific Vulnerability Categorization (SSVC) decision tree [https://www.cisa.gov/sites/default/files/publications/cisa-ssvc-guide%20508c.pdf](https://www.cisa.gov/sites/default/files/publications/cisa-ssvc-guide%20508c.pdf)
+Provide the enumeration of possible decisions in the
```

Comment:
To have the link in its own line and when there's an update in the link, the change set will be limited to the line with the link.

It will be rendered like this.

<img width="665" alt="Screenshot 2024-08-07 at 11 29 43" src="https://github.com/user-attachments/assets/2e728941-4fdb-4e0b-867b-eb370d999e13">

https://bact.github.io/spdx-spec/v3.0.1/model/Security/Properties/decisionType/